src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -285,10 +308,19 @@ public async Task SynchronizeWithServer_ConnectedModeWithIssues_IssuesAddedToSto
             var analysisInformation = new AnalysisInformation("some branch", DateTimeOffset.Now);
 
             var sonarServerMock = CreateSonarService();
-            SetupTaintIssues(sonarServerMock, serverIssue1, serverIssue2);
-            SetupAnalysisInformation(sonarServerMock, analysisInformation);
 
-            var testSubject = CreateTestSubject(taintStore.Object, converter.Object, sonarService: sonarServerMock.Object);
+            var serverBranchProvider = CreateServerBranchProvider("a branch");
+            var bindingConfig = CreateBindingConfig(SonarLintMode.Connected, "projectKey123");
+            SetupTaintIssues(sonarServerMock, "projectKey123", "a branch", serverIssue1, serverIssue2);
+            SetupAnalysisInformation(sonarServerMock, "projectKey123", analysisInformation);
```

Comment:
The test depends on a particular project key and branch, so it needs to specify them explicitly.